spec/services/proofing/resolution/progressive_proofer_spec.rb

```diff
@@ -456,5 +454,78 @@
         end
       end
     end
+
+    # rubocop:disable Layout/LineLength
+    context 'user_can_pass_after_state_id_check is called if same_address_as_id is not on the applicant pii' do
+      # rubocop:enable Layout/LineLength
+      let(:ipp_enrollment_in_progress) { false }
+      let(:applicant_pii) do
+        JSON.parse(<<-STR, symbolize_names: true)
```

Comment:
maybe we can use `MOCK_IDV_APPLICANT` here - but i think we'd have to update it to remove `same_address_as_id = nil`

Reply:
OK good idea. I'd like to use an existing mock rather than redefine a ton of pii just for one test.